python/src/uagents/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .agent import Agent, Bureau  # noqa
 from .context import Context  # noqa
-from .models import Model  # noqa
+from .models import Model, Field  # noqa
```

Comment:
Field v2 are not json serializable anymore
https://docs.pydantic.dev/latest/migration/#changes-to-pydanticfield

Reply:
This most probabably implies that all of integrations and any other agents using Field would be broken with pydantic v2

Reply:
Fields in v2 are different but should still be serialisable. What this tells me is that we have been using `Field` outside of its intended purpose.
By properly handling and adding metadata to the messages we should be able to avoid the use of Field altogether.
This can of cause happen down the line, it's just something to keep in mind.